pkgs/tools/compression/zstd/default.nix

```diff
@@ -59,4 +60,15 @@ stdenv.mkDerivation rec {
     platforms = platforms.unix;
     maintainers = with maintainers; [ orivej ];
   };
-}
+} // stdenv.lib.optionalAttrs static {
+  nativeBuildInputs = [ cmake ];
```

Comment:
It would be good if we can use cmake on non-static as well.

Reply:
Done in the second commit.